pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -154,6 +156,20 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 			"To unset, specify the annotation name followed by a \"-\" (e.g., name-).")
 	p.markFlagMakesRevision("annotation")
 
+	command.Flags().StringArrayVarP(&p.AnnotationsService, "annotation-service", "", []string{},
+		"Service annotation to set. name=value; you may provide this flag "+
+			"any number of times to set multiple annotations. "+
+			"To unset, specify the annotation name followed by a \"-\" (e.g., name-). This flag takes "+
+			"precedence over \"annotation\" flag.")
```

Comment:
```suggestion
			"precedence over the \"annotation\" flag.")
```

Reply:
If you could add the missing article also for the labels doc, that would be awesome.